devicetypes/smartthings/nyce-motion-sensor.src/nyce-motion-sensor.groovy

```diff
@@ -13,6 +13,7 @@
  *  for the specific language governing permissions and limitations under the License.
  *
  */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
```

Comment:
Ah, I see what you mean.  I just checked and if you use def instead of ZoneStatus then the import is not necessary.  However, I think for our example DTHs we should make it clear what is happening and what code is being used so developers can understand how it's working and if they choose not to in their own code, that is their prerogative.